docs/en/ingest-management/elastic-agent/example-kubernetes-fleet-managed-agent-helm.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[example-kubernetes-fleet-managed-agent-helm]]
+= Example: Install {fleet}-managed {agent} on Kubernetes using Helm
+
+preview::[]
+
+This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather Kubernetes metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
```

Comment:
```suggestion
This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather {k8s} metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
```

Reply:
Global search and replace: Use {k8s} variable for Kubernetes

Reply:
Thanks! I've made this change but not globally (excluded UI strings that include "Kubernetes" and of course commands).